source/slang/slang-diagnostics.h

```diff
@@ -99,8 +99,6 @@ namespace Slang
     void printDiagnosticArg(StringBuilder& sb, TokenType tokenType);
     void printDiagnosticArg(StringBuilder& sb, Token const& token);
     void printDiagnosticArg(StringBuilder& sb, CodeGenTarget val);
-    void printDiagnosticArg(StringBuilder& sb, Stage val);
-    void printDiagnosticArg(StringBuilder& sb, ProfileVersion val);
```

Comment:
Is there a reason for moving this logic, which is all about printing of information in diagnostic messages, out of the diagnostics system and over to the profile stuff?

Reply:
Yes - it removed the slang-profile.cpp dependency from DiagnosticSink. 

Which admittedly was only desirable so as to make the amount of dependencies to the internals of Slang source minimal.